datafusion/physical-plan/src/aggregates/group_values/multi_group_by/bytes.rs

```diff
@@ -62,6 +64,11 @@ where
             buffer: BufferBuilder::new(INITIAL_BUFFER_CAPACITY),
             offsets: vec![O::default()],
             nulls: MaybeNullBufferBuilder::new(),
+            max_buffer_size: if O::IS_LARGE {
```

Comment:
I couldn't find any other way to get the max value of the offset size, which is weird. Maybe it would be a nice addition to arrow-rs

Reply:
Agreed: https://github.com/apache/arrow-rs/issues/7474

Reply:
👍